docs/command/mongocli-auth-logout.txt

```diff
@@ -0,0 +1,68 @@
+.. _mongocli-auth-logout:
+
+====================
+mongocli auth logout
+====================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Log out the CLI.
```

Comment:
Log out of MongoDB Atlas.

Reply:
what about something generic, this will log out from wither Atlas or Cloud Manager, whichever you happen to be logged in to?

Reply:
`Log out of the CLI` then.

Reply:
updated